cmd/process-agent/main_common.go

```diff
@@ -169,7 +136,8 @@ Exiting.`
 
 func runAgent(exit chan struct{}) {
 	if opts.version {
-		fmt.Print(versionString("\n"))
+		fmt.Println("WARNING: --version is deprecated and will be removed in a future version. Please use `process-agent version` instead.")
```

Comment:
Small suggestion - may be interesting to add color terminal output here and in another place with deprecation warning, we already have `fatih/color` in the core agent.

Reply:
Love it! Added 😄